### PR TITLE
Add support for cluster_id field for trigger creation

### PR DESCRIPTION
**Pull request by Tetrergeru**
Add field `cluster_id` for trigger creation model to support multiple clusters

List of available clusters can be seen in api at `https://your-moira-url/api/config`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
+# 4.2.0
```
Comment (kissken) [inline CHANGELOG.md:1]: lets add desc and add into doc info about finding cluster id

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+
+Add support for multiple clusters in trigger creation
+
+Now you can use `cluster?id` field in api to set non-default cluster for remote triggers
```
Comment (kissken) [inline CHANGELOG.md]: ?

File: CHANGELOG.md (hunk 1 above)
Comment (Tetrergeru) [reply CHANGELOG.md:1]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (Tetrergeru) [reply CHANGELOG.md]: Fixed

**Review by kissken — APPROVED** [review]

**Review by almostinf — APPROVED** [review]

**Pull request merged**